s/elements/context/element.ts

```diff
@@ -21,7 +22,9 @@ export const NubContext = element<{
 	}).render(use => {
 
 	const [actions] = use.state(stateForActions)
-	const [bindings] = use.state<Bindings>(use.element.bindingsJson ?? defaultBindings)
+	const [bindings, setBindings] = use.state<Bindings>(loadLocalStorageBindings() ?? use.element.bindingsJson ?? defaultBindings)
```

Comment:
i like this `loadLocalStorageBindings() ?? use.element.bindingsJson ?? defaultBindings`

technically, this expression is evaluated on every render. that's too often.

we can tweak this line to improve peformance, like this:

```ts
const [bindings, setBindings] = use.state<Bindings>(
  () => loadLocalStorageBindings() ?? use.element.bindingsJson ?? defaultBindings
)
```

now it's using a state initializer function, which will only be executed one time to find the initial value